sros2/sros2/api/__init__.py

```diff
@@ -279,13 +279,15 @@ def create_key(keystore_path, identity):
     key_dir = os.path.join(keystore_path, relative_path)
     os.makedirs(key_dir, exist_ok=True)
 
-    # copy the CA cert in there
     keystore_ca_key_path = os.path.join(keystore_path, 'ca.key.pem')
     keystore_ca_cert_path = os.path.join(keystore_path, 'ca.cert.pem')
-    dest_identity_ca_cert_path = os.path.join(key_dir, 'identity_ca.cert.pem')
-    dest_permissions_ca_cert_path = os.path.join(key_dir, 'permissions_ca.cert.pem')
-    shutil.copyfile(keystore_ca_cert_path, dest_identity_ca_cert_path)
-    shutil.copyfile(keystore_ca_cert_path, dest_permissions_ca_cert_path)
+
+    # symlink the CA cert in there
+    public_certs = ['identity_ca.cert.pem', 'permissions_ca.cert.pem']
+    for public_cert in public_certs:
+        dst = os.path.join(key_dir, public_cert)
+        relativepath = os.path.relpath(keystore_ca_cert_path, key_dir)
+        os.symlink(src=relativepath, dst=dst)
```

Comment:
Please apply the same try/catch logic as in keymint to avoid raising when updating a keystore: https://github.com/keymint/keymint_tools/blob/9f5bda35dae4ddc927a7f70f2d961451c150f7c0/keymint_tools/build_types/keymint_ros2_dds.py#L171-L176